src/cards/community/Deimos.ts

```diff
@@ -0,0 +1,85 @@
+import { Colony, IColony } from '../../colonies/Colony';
+import { Player } from '../../Player';
+import { Game } from '../../Game';
+import { ColonyName } from '../../colonies/ColonyName';
+import { TileType } from '../../TileType';
+import { AresHandler } from '../../ares/AresHandler';
+import { SelectSpace } from '../../inputs/SelectSpace';
+import { ISpace } from '../../ISpace';
+import { LogHelper } from '../../components/LogHelper';
+import { MAX_COLONY_TRACK_POSITION } from '../../constants';
+
+export class Deimos extends Colony implements IColony {
+    public name = ColonyName.DEIMOS;
+    public description: string = "Hazards";
+
+    public trade(player: Player, game: Game, usesTradeFleet: boolean = true): void {
+        if (usesTradeFleet) this.beforeTrade(this, player, game);
+
+        let steps = 0;
+
+        if (this.trackPosition === MAX_COLONY_TRACK_POSITION) {
+            steps = 3;
+        } else if (this.trackPosition > 3) {
+            steps = 2;
+        } else if (this.trackPosition > 1) {
+            steps = 1;
+        }
+
+        for (let i = 0; i < steps; i++) {
+            const availableSpaces = AresHandler.getAllLandSpacesAdjacentToHazards(game);
+
+            if (availableSpaces.length > 0) {
+                game.interrupts.push({
+                    player: player,
+                    playerInput: new SelectSpace(
+                        'Select space adjacent to hazard tile to erode',
```

Comment:
That's nasty! Ha ha ha ha. Not too nasty?
By the way, I'm probably removing the code that removes space.bonus on overlay because it's incompatible with Dry Desert.

Reply:
Seems ok imo. For trading you get perhaps 2-4 plants / steel / card in the average case. The bonuses are removed from the board to prevent abuse (imagine the draw 3 card / 2 titanium spots etc.)